packages/test/test-version-utils/package.json

```
@@ -27,7 +27,8 @@
     "lint": "npm run eslint",
     "lint:fix": "npm run eslint:fix",
     "test": "npm run test:mocha",
-    "test:mocha": "mocha --unhandled-rejections=strict  --recursive dist/test/*.spec.js --exit --project src/test/tsconfig.json",
+    "test:mocha": "mocha --unhandled-rejections=strict  --recursive dist/test/*.spec.js --exit --project src/test/tsconfig.json -r node_modules/@fluidframework/mocha-test-setup",
```

```suggestion
    "test:mocha": "mocha --unhandled-rejections=strict  --recursive dist/test/*.spec.js --exit --project src/test/tsconfig.json -r @fluidframework/mocha-test-setup",
```

Does this command work with the `node_modules` there? I'm almost certain `mocha-test-setup` gets hoisted to the root of the repo and thus the original version of this script won't find it, but without the relative path it will work thanks to node's package resolution algorithm. It doesn't play well when `npm link`ing packages by hand, but I think it should be the default setup.

---

After having a quick look around the repo, it looks like all of them are configured the way I have it rather than the way you suggested. This is also directly requested as the configuration by fluid-build so I think it should be kept this way?

---

Huh, interesting... the `test:benchmark:report` script I added to `packages/dds/tree` and `experimental/dds/tree` use the approach without node_modules, I remember having issues with them with it there. But you're right that everything else uses the folder, so let's keep it.